### PR TITLE
Add an application to submit and validate an update to a model parameter and corresponding metadata to SimPipe

### PR DESCRIPTION
**Pull request by GernotMaier**
Add  a new application called `submit_model_parameter_from_external.py` to submit a new model parameter to SimPipe.

The new application:

- receives value(s) of a model parameter through command line
- validates input value(s) against schema file for the corresponding model parameter
- validate and enriches metadata obtained (incomplete: see issue #1230)
- write a simtools-db-type json file with the model parameter (ready to be submitted to the DB).

Usage example:

```
     simtools-submit-model-parameter-from-external \
            --parameter num_gains \\
            --value 2 \\
            --instrument LSTN-design \\
            --site North \\
            --model_version 6.0.0 \\
            --input_meta num_gains.metadata.yml
```

This application is intended to be used for the simple case of setting model parameters which are e.g., specifications - meaning no derivation steps are required.

Note that we expect the integration test `test_commandline_tools ` to fail, as we added a new application (see issue #1229).

### CODE REVIEW

**Review by EshitaJoshi — COMMENTED** [review]
Minor comments, but on the whole the code seems okay to me.
Just wanted to ask if it is generally a good idea to give all users an easy way to make direct changes to the database. Is there somewhere where this change gets approved or reviewed? (I assume that's done in the next step of the pipeline?)

File: simtools/data_model/model_data_writer.py [hunk 1]
```diff
@@ -191,15 +198,27 @@ def get_validated_parameter_dict(self, parameter_name, value, instrument, model_
         self._logger.debug(f"Getting validated parameter dictionary for {instrument}")
         schema_file = self._read_model_parameter_schema(parameter_name)
 
+        try:  # e.g. instrument is 'North"
+            site = names.validate_site_name(instrument)
```
[the remark this answers is not in the document]
Comment (GernotMaier) [reply simtools/data_model/model_data_writer.py:202]: We have parameters which are defined for a 'site' and not for a telescopes. These are e.g. the parameter describing the atmosphere. For these, the 'instrument' is the site. This means what with

```
        try:  # e.g. instrument is 'North"
            site = names.validate_site_name(instrument)
        except ValueError:  # e.g. instrument is 'LSTN-01'
            site = names.get_site_from_array_element_name(instrument)
```

we make that both cases are covered (e.g., "North" for a site parameter or "LSTN-01" for a telescope parameter).

**Comment by GernotMaier** [comment]
Thanks for the review! Let me know if the changes are ok!

On the DB access questions:

- we will have different access modes for users, maintainers, developers for the DB (read only, read/write, etc)
- the workflow is actually quite different: changes in model parameters are not submitted to the DB but to the model_parameter repository (through a pull request). 
- only the model_parameter repository is supposed to push to the databases. This happens automatically on release. This is not very well documented at https://gitlab.cta-observatory.org/cta-science/simulations/simulation-model/model_parameters#general-setup

**Review by EshitaJoshi — APPROVED** [review]
Thank you for answering my questions, it makes sense to me now! Also, thank you for the PR!

**Pull request merged**